app/javascript/controllers/timeline/timeline_options_controller.js

```diff
@@ -0,0 +1,89 @@
+class TimelineOptionsController {
+  constructor(miqService, url, categories) {
+    this.reportModel = {
+      tl_show: 'timeline',
+      tl_categories: ['Power Activity'],
+      tl_timerange: 'weeks',
+      tl_timepivot: 'ending',
+      tl_result: 'success',
+      tl_range_count: 1,
+      tl_date: new Date(ManageIQ.calendar.calDateTo),
+    };
+
+    this.miqService = miqService;
+    this.url =  url;
+    this.categories = categories;
+
+    this.afterGet  = true;
+    this.dateOptions = {
+      autoclose: true,
+      todayHighlight: true,
+      orientation: 'bottom',
+    };
+    ManageIQ.angular.scope = this;
+    this.availableCategories = categories;
+  }
+
+  eventTypeUpdated = () => {
```

Comment:
`eventTypeUpdated() {` ;)

It's a syntax error to do `property = function literal` in an es6 class, for some reason.